neuromation/cli/job.py

```diff
@@ -877,6 +873,55 @@ def format_fail(job: str, reason: Exception) -> str:
     return job
 
 
+async def _build_volumes(
+    root: Root, input_volumes: Sequence[str], env_dict: Dict[str, str]
+) -> Set[Volume]:
+    VOLUME_SHORTCUTS = {"HOME", "ALL"}
+    detected_shortcuts = VOLUME_SHORTCUTS & set(input_volumes)
+    if len(detected_shortcuts) > 1:
+        msg = ", ".join(f"--volume={shortcut}" for shortcut in detected_shortcuts)
+        raise click.UsageError(f"Cannot use multiple volume shortcuts together: {msg}")
+
+    volumes: Set[Volume] = set()
+    for vol in input_volumes:
+        if vol in VOLUME_SHORTCUTS:
+            if vol == "HOME":
+                volumes.add(
+                    root.client.parse.volume("storage://~:/var/storage/home:rw")
+                )
+                volumes.add(
+                    root.client.parse.volume(
+                        "storage://neuromation/public:/var/storage/neuromation:ro"
+                    )
+                )
+            if vol == "ALL":
+                available = await root.client.users.get_acl(
+                    root.username, scheme="storage"
+                )
+                root_mountpoint = "/var/storage"
```

Comment:
should we extract a constant and reuse throughout?
we could expose an env var to override the value (for the `HOME` as well).

Reply:
Thanks, for now, I moved this variable to the top of the file. In future, if needed, we always can expose it as an env var.